packages/catalog-process/src/routers/EServiceRouter.ts

```diff
@@ -175,6 +176,7 @@ const eservicesRouter = (
         const ctx = fromAppContext(req.ctx);
 
         try {
+          assertContextHasTokenTypeIn(ctx, ["ui"]);
```

Comment:
still a bit sad that we cannot derive this info from the already declared roles ☹️

Reply:
Trying to do it in another PR: #1698